app/models/user.rb

```diff
@@ -3,4 +3,7 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+  has_and_belongs_to_many :tours
```

Comment:
Dùng cách này https://guides.rubyonrails.org/association_basics.html#the-has-many-through-association